apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -141,16 +147,16 @@ def render(element_html, data):
                             )
 
                             if test["max_points"] != test["points"]:
-                                test["results_color"] = "#E70000"
+                                test["results_color"] = Color("incorrect_red")
                                 test["results_icon"] = "fa-times"
                             # Don't consider points for test cases that are 0/0
                             # to be correct. We compare with a string because
                             # `round_value()` returns a string.
                             elif test["max_points"] == "0":
-                                test["results_color"] = "#757575"
+                                test["results_color"] = Color("gray3")
```

Comment:
Not sure if we want a more semantic name for this one; couldn't think of anything that was better than just 'gray3.